app/controllers/users_controller.rb

```diff
@@ -0,0 +1,45 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    # @user = User.new(username: params[:username], email: params[:email], password: params[:password])
```

Comment:
- **_Remove this comment from your code it makes your work look unprofessional_**